content/docs/started/getting-started-minikube.md

```diff
@@ -21,22 +21,52 @@ Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusi
 ##### Ubuntu
 Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads) or [KVM](http://www.linux-kvm.org/).
 
+The KVM2 driver is intended to replace KVM driver. The KVM2 driver is maintained by the minikube team, and is built, tested and released with minikube.
 For installing KVM:
 
 ```
-$ sudo apt-get install libvirt-bin kvm virt-manager virt-viewer python-spice-client-gtk qemu-kvm qemu-system
-$ sudo usermod -aG libvirtd $USER
+# Install libvirt and qemu-kvm on your system, e.g.
+# Debian/Ubuntu (for older Debian/Ubuntu versions, you may have to use libvirt-bin instead of libvirt-clients and libvirt-daemon-system)
+$ sudo apt install libvirt-clients libvirt-daemon-system qemu-kvm
+
+# Add yourself to the libvirt group so you don't need to sudo
+# NOTE: For older Debian/Ubuntu versions change the group to `libvirtd`
+$ sudo usermod -a -G libvirt $(whoami)
+
+# Update your current session for the group change to take effect
+# NOTE: For older Debian/Ubuntu versions change the group to `libvirtd`
+$ newgrp libvirt
+```
+Then install the driver itself:
```

Comment:
I remember there was a reboot step. Do we need to mention it here?

Reply:
I believe the reboot was needed only to ensure that user's new group membership takes effect. That is actually achieved here by the 'newgrp libvirt' command.